packages/Webkul/Paypal/src/Resources/views/checkout/onepage/paypal-smart-button.blade.php

```diff
@@ -14,6 +14,12 @@
     </style>
 
     <script>
+        let messages = {
+            universalError: "Something went wrong!",
```

Comment:
Please avoid static text and try to use translations instead.

Reply:
Yeah, I am doing that right now just for testing purposes added.